### PR TITLE
Debug app start

### PR DESCRIPTION
**Pull request by sanjeevdwivedi**
## Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
Fix app start issues

## Issues Closed or Referenced

- Closes #<issue number> (this will automatically close the issue when the PR closes)
- References #132  (this references the issue but does not close with PR)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -53,6 +53,12 @@
             <version>1.6.4</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure</artifactId>
+            <version>1.23.0</version>
+        </dependency>
+
```
Comment (jyotsnaravikumar) [inline pom.xml:61]: Please ensure all tests pass with this verison

File: src/main/java/com/microsoft/cse/helium/app/services/keyvault/KeyVaultService.java [hunk 2]
```diff
@@ -9,6 +10,8 @@
 import com.microsoft.azure.keyvault.models.KeyBundle;
 import com.microsoft.azure.keyvault.models.SecretBundle;
 import com.microsoft.azure.keyvault.models.SecretItem;
+//import com.microsoft.azure.management.Azure;
+//import com.microsoft.azure.management.keyvault.Vault;
 import com.microsoft.cse.helium.app.Constants;
```
Comment (jyotsnaravikumar) [inline src/main/java/com/microsoft/cse/helium/app/services/keyvault/KeyVaultService.java:15]: is pmd checks letting this? Usually it barfs with ununsed imports

Comment (sanjeevdwivedi) [reply src/main/java/com/microsoft/cse/helium/app/services/keyvault/KeyVaultService.java:15]: :D Yes, as you know, we will have build break if code is not PMD clean. That being said, I just fixed it.

File: pom.xml (hunk 1 above)
Comment (sanjeevdwivedi) [reply pom.xml:61]: fixed, all tests are passing.

**Pull request merged**